### PR TITLE
ikev1: added ikev1 test - v4

### PR DESCRIPTION
**Pull request by fhonza**
This PR adds checks for a new IKEv1 parser for suricata, which is currently in review.
It checks for metadata logging and keywords triggering alerts.

### CODE REVIEW

File: tests/ikev2-weak-dh/test.yaml [hunk 1]
```diff
@@ -31,3 +32,38 @@ checks:
         ikev2.payload[1]: KeyExchange
         ikev2.payload[2]: SecurityAssociation
         ikev2.payload[3]: NoNextPayload
+
+  - filter:
+      count: 1
+      version: 5
```
Comment (catenacyber) [inline tests/ikev2-weak-dh/test.yaml:38]: @inashivb can we do only one filter for versions 4 and 5 ?

File: tests/ikev2-weak-dh/test.yaml [hunk 2]
```diff
@@ -31,3 +32,38 @@ checks:
...
+        ikev2.payload[0]: Nonce
+        ikev2.payload[1]: KeyExchange
+        ikev2.payload[2]: SecurityAssociation
+        ikev2.payload[3]: NoNextPayload
+
+  - filter:
+      count: 1
+      min-version: 6
+      match:
+        event_type: ike
```
Comment (catenacyber) [inline tests/ikev2-weak-dh/test.yaml:57]: @fhonza Can you add a simple comment to say that the only change is renaming the event from ikev2 to generic ike ?

**Comment by catenacyber** [comment]
CI is broken for now for versions 4 and 5 cf https://github.com/OISF/suricata/pull/5440

**Comment by catenacyber** [comment]
Follows #324

File: tests/ikev2-weak-dh/test.yaml (hunk 2 above)
Comment (fhonza) [reply tests/ikev2-weak-dh/test.yaml:57]: yes, will add one...

File: tests/ikev2-weak-dh/test.yaml (hunk 1 above)
Comment (inashivb) [reply tests/ikev2-weak-dh/test.yaml:38]: No, we could check for any version.

**Pull request closed without merging**